pkg/frontend/frontend_test.go

```diff
@@ -0,0 +1,39 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"regexp"
+	"strings"
+	"testing"
+
+	"github.com/gorilla/mux"
+)
+
+func TestURLPathsAreLowerCase(t *testing.T) {
+	f := &frontend{}
+
+	fakeRouter := mux.NewRouter()
+	f.unauthenticatedRoutes(fakeRouter.NewRoute().Subrouter())
```

Comment:
I would be up for seeing the router setup in frontend.Run pulled out into a nonexported method to avoid the duplication here, wdyt?